src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTaskTest.java

```diff
@@ -0,0 +1,256 @@
+import static org.junit.Assert.assertThat;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.testhelper.AssertFilm;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.jsoup.Jsoup;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+
+class TestData {
+  public String requestUrl;
+  public String filmPageFile;
+  public String jsUrl;
+  public String jsFile;
+  public String m3u8Url;
+  public String m3u8File;
+  public String topic;
+  public String expectedTitle;
+  public LocalDateTime expectedDate;
+  public Duration expectedDuration;
+  public String expectedDescription;
+  public String expectedSubtitle;
+  public String expectedUrlSmall;
+  public String expectedUrlNormal;
+  public String expectedUrlHd;
+  public GeoLocations[] expectedGeoLocations;
+
+  public TestData(
+      final String aRequestUrl,
+      final String aFilmPageFile,
+      final String aJsUrl,
+      final String aJsFile,
+      final String aM3u8Url,
+      final String aM3u8File,
+      final String aTopic,
+      final String aExpectedTitle,
+      final LocalDateTime aExpectedDate,
+      final Duration aExpectedDuration,
+      final String aExpectedDescription,
+      final String aExpectedSubtitle,
+      final String aExpectedUrlSmall,
+      final String aExpectedUrlNormal,
+      final String aExpectedUrlHd,
+      final GeoLocations[] aExpectedGeoLocations) {
+    requestUrl = aRequestUrl;
+    filmPageFile = aFilmPageFile;
+    jsUrl = aJsUrl;
+    jsFile = aJsFile;
+    m3u8Url = aM3u8Url;
+    m3u8File = aM3u8File;
+    topic = aTopic;
+    expectedTitle = aExpectedTitle;
+    expectedDate = aExpectedDate;
+    expectedDuration = aExpectedDuration;
+    expectedDescription = aExpectedDescription;
+    expectedSubtitle = aExpectedSubtitle;
+    expectedUrlSmall = aExpectedUrlSmall;
+    expectedUrlNormal = aExpectedUrlNormal;
+    expectedUrlHd = aExpectedUrlHd;
+    expectedGeoLocations = aExpectedGeoLocations;
+  }
+}
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+@PowerMockRunnerDelegate(Parameterized.class)
+@PowerMockIgnore("javax.net.ssl.*")
+public class WdrFilmDetailTaskTest extends WdrTaskTestBase {
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(
+        new Object[][] {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)